x-pack/plugins/apm/public/components/shared/unified_search_bar/index.tsx

```diff
@@ -138,11 +140,12 @@ export function UnifiedSearchBar({
     },
     core,
   } = useApmPluginContext();
-  const { services } = useKibana<ApmPluginStartDeps>();
+  const { services } = useKibana<ApmPluginStartDeps & ApmServices>();
```

Comment:
Perhaps this should take a single type:

```suggestion
  const { services } = useKibana<ApmDependencies>();
```

Is there anywhere we only have access to `ApmPluginStartDeps` and not `ApmServices`?

Reply:
Btw this reminds me of: https://github.com/elastic/kibana/issues/152098